src/memory/client/MemoryBrowser.tsx

```diff
@@ -318,4 +357,36 @@ export class MemoryBrowser extends React.Component<Props, State> {
       </div>
     );
   }
+
+  renderChildName() {
+    if (MemoryBrowser.firtTime) {
+      this.getAvailableChilds();
+      MemoryBrowser.firtTime = false;
```

Comment:
This needs to be reset to true so that future debug sessions fetch the availableChildren again?

Reply:
I just want to `getAvailableChilds` function is called only one time when Memory Browser view is opened